packages/itwinui-react/src/core/Footer/Footer.test.tsx

```diff
@@ -35,7 +35,9 @@ const assertFooterItems = (
     }
 
     const dataIndex = Math.floor(i / 2);
-    expect(element.textContent).toBe(data[dataIndex].title);
+    expect(element.textContent?.replace(' (opens in new tab)', '')).toBe(
+      data[dataIndex].title,
+    );
```

Comment:
This is testing implementation details and is therefore brittle. You can instead do a partial match using `toHaveTextContent`:

```js
expect(element).toHaveTextContent(data[dataIndex].title);
```

Reply:
Alright, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/34d0dad5d00a597dd17d66d6706c4f98dd71c73f)